VideoLocker/res/values/strings.xml

```diff
@@ -36,8 +36,17 @@
     <string name="course_detail_enroll_now_button">Enroll Now</string>
     <!-- About This Course title -->
     <string name="about_this_course">About this Course</string>
-    <!-- Enroll Now text-->
-    <string name="enroll_now">Enroll Now</string>
+    <!-- Enroll Now button text-->
+    <string name="enroll_now_button_text">Enroll Now</string>
+    <!-- View Course button text text-->
+    <string name="view_course_button_text">View Course</string>
+    <!-- Floating overlay text shown to user when they succeed at enrolling in a course -->
+    <string name="you_are_now_enrolled">You have enrolled in this course</string>
```

Comment:
How about "You are now enrolled in this course". Seems to me that the message should confirm the new state of the system rather than confirming the user's action.